drivers/net/wireless/realtek/rtl8192cu/core/rtw_security.c

```diff
@@ -1878,8 +1878,8 @@ _func_enter_;
     for (j = 0; j < 8; j++)
     	message[payload_index+j] = mic[j];
 
-	payload_index = hdrlen + 8;
-	for (i=0; i< num_blocks; i++)
+    payload_index = hdrlen + 8;
+    for (i=0; i< num_blocks; i++)
```

Comment:
I'd suggest dropping this commit. We are now using unmodified code from:  https://github.com/pvaret/rtl8192cu-fixes
If this is wanted, it could be submitted there and we'll pick it up.

Reply:
Done.